### PR TITLE
Fix domain status history view

### LINKED ISSUE #834: Admin: domain history - 3 identical records created on accepting registrant change request
steps to reproduce:
* create a request to change registrant of a domain
* click on the link that was sent to current registrant and accept the change
* go to admin find the domain in the list of domains (/admin/domains) and click it open
* click on history button to open domain_history view (/admin/domains/86931/domain_versions)
* you will see 4 new records - 1. for the registrant change request and then 3 identical records for the accepting

expected result is 2 new records - for the request and for the actual change.

### PR DESCRIPTION
**Pull request by maricavor**
Related to #834

Fixed duplication of records in domain status history after status change by admin.

Added pagination for domain versions with per_page = 10
Added I18n pluralization for result_count

### CODE REVIEW

File: app/controllers/admin/domains_controller.rb [hunk 1]
```diff
@@ -63,8 +63,11 @@ def update
     # rubocop:enable Metrics/MethodLength
 
     def versions
+      per_page = 10
```
Comment (yulgolem) [inline app/controllers/admin/domains_controller.rb:66]: Looks like some magic number to me. Consider using a constant like DEFAULT_RECORDS_PER_PAGE and defining it in the controller itself.

Comment (maricavor) [reply app/controllers/admin/domains_controller.rb:66]: Thanks! Just saw the same apporach in other controllers. Actually I wanted to add paginates_per 10 to the model itself, but did not want to make it general rule. I will do like you suggested - add constant in the controller!

File: app/controllers/admin/domains_controller.rb [hunk 2]
```diff
@@ -65,6 +67,8 @@ def update
     def versions
       @domain = Domain.where(id: params[:domain_id]).includes({ versions: :item }).first
       @versions = @domain.versions
+      @last_version = @versions.last
+      @old_versions = Kaminari.paginate_array(@versions.not_creates.reverse).page(params[:page]).per(DEFAULT_VERSIONS_PER_PAGE)
```
Comment (yulgolem) [inline app/controllers/admin/domains_controller.rb]: Do our user have any ability to input a number of entries per page? If so I'd suggest to use something like 
params[:per_page] || DEFAULT_VERSIONS_PER_PAGE , maybe moved to some helper method.
If not- ok then, we are good to go

**Review by yulgolem — APPROVED** [review]

File: app/controllers/admin/domains_controller.rb (hunk 2 above)
Comment (maricavor) [reply app/controllers/admin/domains_controller.rb]: No, there is no per page input, not necessary

**Review by OlegPhenomenon — APPROVED** [review]

**Review by DinSmol — APPROVED** [review]

**Review by DinSmol — APPROVED** [review]

**Pull request merged**